application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -227,6 +227,8 @@ public function checkTags(Validation $validation, $tags)
 
 	public function checkValues(Validation $validation, $attributes, $fullData)
 	{
+
+		$attributes = !empty($fullData['values']) ? $fullData['values'] : [];
```

Comment:
I'm stepping through this now and it seems as though when the value is an array of strings it isn't being passed correctly to the function in the validator.